web/src/javascript/app.js

```diff
@@ -3,15 +3,26 @@ import {drawGame, createGameCanvas} from "./game";
 import {gameRefresh} from "./constants";
 
 const gameCanvas = createGameCanvas();
-let currentState;
 
-(new Worker("javascript/gameWebWorker.bundle.js")).onmessage = message => currentState = message.data;
+let currentState;
+const networkWorker = new Worker("javascript/gameWebWorker.bundle.js");
+networkWorker.onmessage = message => {
+  // This is to prevent Chrome's GC from deleting the worker.
```

Comment:
Isn't it the `const networkWorker = ...` that's there to prevent the GCing? If that is the case, why is the comment in the `onmessage` function? It makes it feel like the *if* is the way that we avoid the problem, rather than by assigning it to a variable.